src/main/app/src/app/shared/edit/edit-datum-groep/edit-datum-groep.component.ts

```diff
@@ -298,6 +298,10 @@ export class EditDatumGroepComponent extends EditComponent implements OnChanges,
         }
     }
 
+    get isHervat() {
```

Comment:
Je geeft een string terug. Zet het return type van de method op boolean

Reply:
opgelost in opvolgende commit